icu4c/source/common/rbbisetb.h

```diff
@@ -133,6 +133,7 @@ class RBBISetBuilder : public UMemory {
     int32_t               fGroupCount;
 
     UBool                 fSawBOF;
+    UBool                 fUse8Bits;
```

Comment:
unused? the .cpp file puts this into local variables.

Reply:
good catch!